libnmstate/netapplier.py

```diff
@@ -140,6 +141,7 @@ def _apply_ifaces_state(
                     ifaces2add + ifaces2edit,
                     con_profiles=ifaces_add_configs + ifaces_edit_configs,
                 )
+            _disable_ipv6(desired_state)
```

Comment:
Please save the current ipv6 state of all interfaces, so that we could rollback on failure.

Reply:
Is expect the rollback to succeed when moving from `enable` to `disable`.. NM should manage to rollback to `enable`.

The other way around may not work, but we can mark it as a limitation for now (until NM provides the solution). I guess we can manage it as well, but just for this small think I would prefer to avoid it if possible.

I will add the relevant tests to check this out and reference the limitation if they are seen.